### PR TITLE
updating documentation

### PR DESCRIPTION
**Pull request by Casyfill**

### CODE REVIEW

File: docs/creating_your_tooltip.md [hunk 1]
```diff
@@ -85,7 +88,7 @@ You can create your tooltip using [`vegaTooltip`](APIs.md#tooltip). This functio
 vegaEmbed("#vis", spec, opt)
   .then(function(result) {
     // result.view is the Vega View
-    vegaTooltip(result.view);
+    vegaTooltip.default(result.view);
```
Comment (domoritz) [inline docs/creating_your_tooltip.md]: Default is only necessary if you don't use a bundler as I explain below. I'm not sure what the cleanest way to explain this is.

File: docs/creating_your_tooltip.md [hunk 2]
```diff
@@ -65,8 +65,11 @@ Alternatively, you can create a runtime dataflow of the visualization using [`ve
 
 ```js
 var runtime = vega.parse(spec);
+var tooltip = new vegaTooltip.Handler(options);
```
Comment (domoritz) [inline docs/creating_your_tooltip.md]: This snippet intentionally doesn't add the tooltips yet. Can you explain why you think it should add a tooltips?

Comment (Casyfill) [reply docs/creating_your_tooltip.md]: oh, perhaps I misunderstood that

File: docs/creating_your_tooltip.md (hunk 1 above)
Comment (Casyfill) [reply docs/creating_your_tooltip.md]: ah, ic - I think this note wasn't there yesterday... feel free to drop this PR then. 
Perhaps there is a way to highlight this note in the markdown? spent some time figuring that out

**Comment by Casyfill** [comment]
On the separate notice, it would be great to have a table / multi-feature tooltip example

File: docs/creating_your_tooltip.md (hunk 1 above)
Comment (domoritz) [reply docs/creating_your_tooltip.md]: Thanks for the PR! I think this is a bit confusing. You can highlight in markdown for example with `**`.

**Comment by domoritz** [comment]
The examples are in https://github.com/vega/vega-tooltip/tree/master/exampleSpecs

**Pull request closed without merging**